pylti/flask.py

```diff
@@ -327,7 +327,7 @@ def close_session():
         session[LTI_SESSION_KEY] = False
 
 
-def lti(app=None, request='any', error=default_error, role='any',
+def lti(app=None, request='any', error=default_error, role='any', url=None,
         *lti_args, **lti_kwargs):
     """
```

Comment:
I added a comment as requested. Hopefully the comment will answer your question.